File: snippets/csharp/SendPayment.cs
using Breez.Sdk;

public class SendPaymentSnippets
{
    public void SendPayment(BlockingBreezServices sdk)
    {
        // ANCHOR: send-payment
        var bolt11 = "...";
        var useTrampoline = false;
        ulong optionalAmountMsat = 3_000_000;
        var optionalLabel = "<label>";
        
        
        try
        {
            // The `amountMsat` param is optional and should only passed if the
            // bolt11 doesn't specify an amount.
            // The amountMsat is required in case an amount is not specified in
            // the bolt11 invoice.
            var response = sdk.SendPayment(
                new SendPaymentRequest(bolt11, useTrampoline, optionalAmountMsat, optionalLabel));
        }
        catch (Exception)
        {
            // Handle error
        }
        // ANCHOR_END: send-payment
    }
}

File: snippets/python/src/send_payment.py
import breez_sdk


def send_payment(sdk_services):
    # ANCHOR: send-payment
    bolt11 = "..."
    try:
        use_trampoline = False
        # The `amount_msat` param is optional and should only passed if the bolt11 doesn't specify an amount.
        # The amount_msat is required in case an amount is not specified in the bolt11 invoice'.
        optional_amount_msat = 3000000
        optional_label = "<label>"
        req = breez_sdk.SendPaymentRequest(bolt11=bolt11, use_trampoline=use_trampoline, amount_msat=optional_amount_msat, label=optional_label)
        sdk_services.send_payment(req=req)
    except Exception as error:
        print(error)
        raise        
    # ANCHOR_END: send-payment
